core/src/com/fallenflame/game/GameEngine.java

```diff
@@ -210,15 +212,15 @@ public void dispose() {
     public void reset() {
         level.dispose();
 
-        setComplete(false);
-        setFailure(false);
-        countdown = -1;
+        setIsSuccess(false);
+        setIsFailed(false);
+        // countdown = -1; TODO Laura: what is this for?
```

Comment:
This one added time to put a "you died" or a "you won" message before it reset. I wasn't sure if I wanted to keep it in or not, but looking at this more this seems necessary

Reply:
I'll add support in a separate branch/pr